google/cloud/bigtable_v2/services/bigtable/transports/pooled_grpc_asyncio.py

```diff
@@ -0,0 +1,578 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
This file is autogenerated from the [template in our gapic-generator fork](https://github.com/googleapis/gapic-generator-python/blob/bigtable_gapic/gapic/templates/%25namespace/%25name_%25version/%25sub/services/%25service/transports/pooled_grpc_asyncio.py.j2)

It uses the standard async grpc transport as a base, with modifications to support a pool of channels, instead of just one